src/components/AlertModal/AlertModal.tsx

```diff
@@ -40,7 +40,16 @@ const alertModalBodyIconStyle = css`
   margin-right: ${spacing.small};
 `;
 
-const AlertModal = ({ text, onCancel, actions, component, show, severity }) =>
+interface Props {
+  text: string;
+  onCancel: () => void;
+  component?: React.ReactElement[] | React.ReactElement;
+  actions?: { text: string; onClick: Function }[];
```

Comment:
Bør onClick være mer spesifikk enn Function?

Reply:
Egentlig, ja. Problemet er at komponenten til tider kalles med Redux onclicks med litt spesifikke typer. Da ga det mer mening for meg å ikke spesifisere akkurat hvilke funksjonssignaturer man godtok.